### DISCUSSION TITLE
spotted a few typos in the docs 👀

### DISCUSSION THREAD

**Discussion by goffxnca** [bisq-network/bisq-website#501] [General]
i was going through the docs and saw a few typos. here’s one i took screenshot of <img width="927" alt="image" src="https://github.com/user-attachments/assets/d9b07e08-4d37-4c40-839f-b5689d31fad6">

figured i’d go through and clean them up and put together a pr if that works!

**Comment by cbeams** [comment]
Hi @goffxnca, yes, please go ahead. Typo fixes are a great way to get started.

**Comment by goffxnca** [comment]
Ref to the merged PR #502 (GitHub doesn’t auto-link)
